### ISSUE TITLE
E523: Not allowed here

### ISSUE THREAD

**Issue by kkoomen** [arthurxavierx/vim-caser#8]
I've noticed that I can do : `gs_` and then `viW`, but I can't select a word first with `viW` and then do `gs_`. 

It says: 

```
Error detected while processing function <SNR>23_DoAction:
line   31:
E523: Not allowed here
```

any idea?

My vim version is 8.1.1850

**Comment by mulias** [comment]
Same issue, tried with neovim 3.8 and 4.2.

**Comment by mulias** [comment]
Also the same issue as https://github.com/arthurxavierx/vim-caser/issues/6

**Comment by arthurxavierx** [comment]
Fixed in #10.
